src/components/input-field/input-field.tsx

```diff
@@ -624,4 +625,17 @@ export class InputField {
             this.action.emit();
         }
     }
+    private handleWheel() {
+        if (this.type === 'number') {
+            // In some browsers (Chrome for example), hovering the input with
+            // the input focused, and scrolling, will both change the value
+            // AND scroll the page. We would prefer to never change the value
+            // on scroll, instead always scrolling the page, but since we
+            // haven't found a way to do that, this is the next best thing, as
+            // it prevents the page from being scrolled, but only in the
+            // circumstances when the value is changed by the scrolling.
+            // Please test THOROUGHLY if you remove it 😄
```

Comment:
Maybe remove this last line before we merge :smile:

Reply:
No, that's meant to be left in, in case anyone gets the idea to fiddle with this sometime in the future…

But maybe the whole comment needs some improvement 🙂

Reply:
Oh yeah right, I read it as something completely different haha :smile: